### PR TITLE
add login menu

### PR DESCRIPTION
**Pull request by mishanika**

### CODE REVIEW

File: src/containers/login/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  const renderFooter = ({ text, url }: IFooter) => (
+    <div className="footer-element" onClick={() => window.open(url)}>
+      <a href="#" className="footer-a">
+        {text}
+      </a>
+    </div>
+  );
+  const googlePlayRedirect = () => {
+    window.open('https://play.google.com/store/apps/details?id=com.instagram.android');
+  };
```
Comment (Io38) [inline src/containers/login/Login.tsx:25]: отступ

File: src/containers/login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  const microsoftRedirect = () => {
+    window.open(
+      'ms-windows-store://pdp/?productid=9nblggh5l9xt&referrer=appbadge&source=www.instagram.com&mode=mini&pos=0%2C0%2C3052%2C1966',
+    );
+  };
```
Comment (Io38) [inline src/containers/login/Login.tsx:30]: отступ

File: src/containers/login/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  useEffect(() => {
+    const phoneBlock = document.querySelectorAll('.screenshot');
+    let counter = 0 as keyof typeof phoneBlock as number;
+    setInterval(() => {
```
Comment (Io38) [inline src/containers/login/Login.tsx:34]: always clear intervals and event listenters

File: src/containers/navigation/routes.tsx [hunk 4]
```diff
@@ -1,29 +1,35 @@
...
-import Reels from "../reels/Reels";
-import Direct from "../direct/Direct";
-import Profile from "../profile/Profile";
-import { IRoute } from "./types";
+import Main from '../main/Main';
+import Explore from '../explore/Explore';
+import Reels from '../reels/Reels';
+import Direct from '../direct/Direct';
+import Profile from '../profile/Profile';
+//import Registration from '../login/Login';
```
Comment (Io38) [inline src/containers/navigation/routes.tsx:6]: delete comment

**Review by Io38 — CHANGES_REQUESTED** [review]

**Review by mishanika — COMMENTED** [review]
image
image
image

**Pull request merged**